packages/types/src/core/core.ts

```diff
@@ -19,6 +19,7 @@ export declare namespace CoreTypes {
     keychain?: IKeyChain;
     storage?: IKeyValueStorage;
     storageOptions?: KeyValueStorageOptions;
+    customStoragePrefix?: string;
```

Comment:
This would've been nice to be included `storageOptions?` but might be difficult to achieve

Reply:
Yeah I see that in terms of opts here. But that would create other aspects that don't really make sense, i.e. the `KeyValueStorageOptions` are specifically consumed by the `KeyValueStorage` class, but these keys are actually set by the controllers, not `KeyValueStorage`.